Tests/Observable+MoyaSpec.swift

```diff
@@ -266,5 +266,87 @@ class ObservableMoyaSpec: QuickSpec {
                 expect(receivedError).to(beOfSameErrorType(expectedError))
             }
         }
+
+        describe("object mapping") {
```

Comment:
Can we add a test that maps an array of objects?

Reply:
Sure! Added.